BoxContentSDK/BoxContentSDK/Models/BOXFile.m

```diff
@@ -37,6 +37,22 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
                                                    inDictionary:JSONResponse
                                                 hasExpectedType:[NSString class]
                                                     nullAllowed:NO];
+        
+        NSDictionary *fileVersionDictionary = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyFileVersion
+                                                                            inDictionary:JSONResponse
+                                                                         hasExpectedType:[NSDictionary class]
+                                                                             nullAllowed:YES
+                                                                       suppressNullAsNil:YES];
+        // Parse File Version.
+        if (fileVersionDictionary)
+        {
+            self.fileVersion = [[BOXFileVersionMini alloc] initWithJSON:fileVersionDictionary];
+        }
+        
+        self.fileVersion = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyFileVersion
```

Comment:
Sorry, I meant that we're setting `self.fileVersion` correctly above, and this would set it back to `nil` or crash.

Reply:
Ok, thanks. I removed it.